src/Service.Tests/Snapshots/ConfigurationTests.TestReadingRuntimeConfigForCosmos.verified.txt

```diff
@@ -271,18 +271,22 @@
         },
         Permissions: [
           {
-            Role: anonymous,
+            Role: field-mutation-with-read-permission,
```

Comment:
why not keep `anonymous` role  and add `field-mutation-with-read-permission` as a distinct role? that way you preserve what some tests may be validating.

Reply:
actually with `anonymous` role..test cases were failing over to that role if read permission is not there for Mutation.

Reply:
I can change the entity to something else..to preserve these scenarios. IMO anonymous has nothing special, its just a default role.

Reply:
please refer other comments.